src/query.ts

```diff
@@ -397,6 +397,19 @@ class Query {
    *     If not specified, default values are chosen by Datastore for the
    *     operation. Learn more about strong and eventual consistency
    *     [here](https://cloud.google.com/datastore/docs/articles/balancing-strong-and-eventual-consistency-with-google-cloud-datastore).
+   * @param {object} [options.gaxOptions] Request configuration options, outlined
+   *     here: https://googleapis.github.io/gax-nodejs/global.html#CallOptions.
+   * @param {boolean} [options.wrapNumbers=false]
+   *     Indicates if the numbers should be wrapped in Int wrapper.
```

Comment:
`Wrap values of `integerValue` type in {@Datastore#Int} objects.`

^-- I'm not actually sure if {@Datastore#Int} is the right way to link to that definition?

Reply:
It seems it should be {@link Datastore#Int} at least that's what the JSDoc documentation says